components/Layout.vue

```diff
@@ -139,14 +139,10 @@ const contentPath = path.endsWith('/') ? path.slice(0, -1) : path;
 const contentDirectory = await queryContent().where({ _path: contentPath }).findOne();
 const title = contentDirectory.title;
 const navigation = await fetchContentNavigation();
-const allLinks = navigation.flatMap((section) => section.children);
-const linkIndex = allLinks.findIndex((link) => link._path === path);
-const previousPage = allLinks[linkIndex - 1];
-const nextPage = allLinks[linkIndex + 1];
 const section = navigation.find((s) =>
   s.children.find((child) => child._path === path)
 );
-const { toc } = useContent();
+const { toc, prev: previousPage, next: nextPage } = useContent();
```

Comment:
Any reason why we needed to rename the variables here?

Reply:
These were the names already being used in the Vue template - that's why this diff is small 👍 

I like the idea of [including the section heading titles](https://content.nuxtjs.org/api/composables/use-document-driven) in the prev/next navigation too:

![image](https://user-images.githubusercontent.com/46909693/195752719-f6ee93af-10c7-4199-887e-df907fb817f0.png)

 I imagined these variables as `prevSectionTitle` (Component) and `nextSectionTitle` (Utils) respectively so more descriptive variables felt better here

Reply:
@Ray2126 I've been playing around and I don't think #3 needs to be considered anymore - I've updated the variables to use the names provided by the Nuxt composable.

Please note the new additions to the diff